staging/src/k8s.io/kubectl/pkg/drain/filters.go

```diff
@@ -89,62 +90,72 @@ func (l *podDeleteList) errors() []error {
 	return errs
 }
 
-type podDeleteStatus struct {
+// PodDeleteStatus informs filters if a pod should be deleted
+type PodDeleteStatus struct {
 	delete  bool
```

Comment:
Don't you have to make these public as well to use these from outside this package?

Reply:
Probably.  I rely on the linter to tell me these things, but it didn't mention it for this.  I'll make it public.

Reply:
Yeah, we have some factory functions for people to use, so the internal fields don't need to be public, but making them public might aid in testing and development of the end user.